files/src/extension.cpp

```diff
@@ -350,15 +350,19 @@ void Files::Extension::handleQuery(Core::Query * query) const {
                     auto item = make_shared<StandardItem>(fileInfo.filePath(),
                                                           icon,
                                                           fileName,
-                                                          fileInfo.filePath());
+                                                          fileInfo.filePath(),
+                                                          dir.filePath(fileName));
                     item->setActions(File::buildFileActions(fileInfo.filePath()));
-                    items.push_back(move(item));
+                    results.emplace_back(move(item), 0);
```

Comment:
Why are we switching from `push_back` to `emplace_back`?
AFAIK `emplace_back` is more efficient only when the instance is created in place.

Reply:
When I was writing the changes, push back wouldn't work. I haven't tested to change it back since then but the items I pushed just didn't show up.